### PR TITLE
docs: refactor image to image example

### PR DESCRIPTION
**Pull request by jupyterjazz**
Part of https://github.com/jina-ai/finetuner/issues/451
Closes https://github.com/jina-ai/finetuner/issues/453

Refactor image to image fine-tuning example based on the new template

### CODE REVIEW

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 1]
```diff
@@ -17,30 +17,32 @@ The dataset consists of pairs of images, these are the positive pairs. Negative
...
 
-## Preparing data
-Training and evaluation data are already prepared and pushed to Hubble following the {ref}`instructions <create-training-data>`.
-You can either pull the data:
-```python
-from docarray import DocumentArray
-train_data = DocumentArray.pull('resnet-ttl-train-data')
-eval_data = DocumentArray.pull('resnet-ttl-eval-data')
+## Data
+Out journey starts locally. We have to {ref}`prepare the data and push it to the cloud <create-training-data>` and Finetuner will be able to get the data by its name. For this example,
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:21]: Our

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 2]
```diff
@@ -17,30 +17,32 @@ The dataset consists of pairs of images, these are the positive pairs. Negative
...
+:class: tip
+We don't require to push data to the cloud by yourself. Instead of a name, you can provide a `DocumentArray` and Finetuner will do the job for you.
 ```
-Or specify given `DocumentArray` names (`resnet-ttl-train-data` and `resnet-ttl-eval-data`) directly to Finetuner.
 
-## Choosing the model
-Now let's see what backbone models we can use. You can see available models either in {ref}`choose backbone <choose-backbone>` section or by calling `finetuner.describe_models()`.
+
+## Backbone model
+Now let's see which backbone models we can use. You can see available models either in {ref}`choose backbone <choose-backbone>` section or by calling [`finetuner.describe_models()`](../../api/finetuner/#finetuner.describe_models).
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:31]: I think it's easier to link to the developer reference like this: {meth}`~finetuner.describe_models()`

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 3]
```diff
@@ -17,30 +17,32 @@ The dataset consists of pairs of images, these are the positive pairs. Negative
...
 
 
 For this example, we're gonna go with `resnet50`.
 
-## Creating a fine-tuning job
-You can easily start a fine-tuning run with `finetuner.fit`.
+## Fine-tuning
+From now on, all the action happens in the cloud!
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:37]: ❤️‍🔥

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 4]
```diff
@@ -87,18 +92,11 @@ finetuner.login()
 run = finetuner.get_run('resnet-ttl')
 ```
 
-You can continue monitoring the runs by checking the status - `run.status()` or the logs - `run.logs()`. 
-
-## Saving your model
+You can continue monitoring the runs by checking the status - [`run.status()`](../../api/finetuner.run/#finetuner.run.Run.status) or the logs - [`run.logs()`](../../api/finetuner.run/#finetuner.run.Run.logs).
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:95]: same here, {meth}`~finetuner.run.Run.status()` etc

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 5]
```diff
@@ -87,18 +92,11 @@ finetuner.login()
...
+You can continue monitoring the runs by checking the status - [`run.status()`](../../api/finetuner.run/#finetuner.run.Run.status) or the logs - [`run.logs()`](../../api/finetuner.run/#finetuner.run.Run.logs). 
 
-If your runs have finished successfully, you can save fine-tuned models in the following way:
-```python
-run.save_model('resnet-model')
-```
-
-## Evaluating your model
+## Evaluating
 Currently, we don't have a user-friendly way to get evaluation metrics from the `EvaluationCallback` we initialized previously.
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:98]: also add dev ref for {class}`~finetuner.callback.EvaluationCallback`

**Review by violenil — COMMENTED** [review]
This is looking a lot better! I like how you are guiding from local -> cloud -> local. Left some minor comments.

File: docs/3_finetuner_in_action/3_2_image_to_image.md (hunk 1 above)
Comment (gmastrapas) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:21]: get the dataset by its name

File: docs/3_finetuner_in_action/3_2_image_to_image.md [hunk 6]
```diff
@@ -17,30 +17,32 @@ The dataset consists of pairs of images, these are the positive pairs. Negative
...
-```{admonition} Login to Jina Cloud
-:class: tip
-Before creating a run, you need to {ref}`login to Jina ecosystem <login-to-jina-ecosystem>` by calling `finetuner.login()`.
+First you need to {ref}`login to Jina ecosystem <login-to-jina-ecosystem>`:
+```python
+import finetuner
+finetuner.login()
 ```
 
+Now, you can easily start a fine-tuning job with [`finetuner.fit`]((../../api/finetuner/#finetuner.fit)):
```
Comment (bwanglzu) [inline docs/3_finetuner_in_action/3_2_image_to_image.md:45]: let's reference to the developer reference with `meth`

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Can we add before/after comparison?

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: docs/3_finetuner_in_action/3_2_image_to_image.md (hunk 3 above)
Comment (violenil) [reply docs/3_finetuner_in_action/3_2_image_to_image.md:37]: The guiding is great!

**Review by violenil — APPROVED** [review]
LGTM! 🚀

**Pull request merged**